css/src/atomics/width.scss

```diff
@@ -0,0 +1,101 @@
+$widths: 100, 150, 200, 250, 300, 350, 400, 450, 500 !default;
+// do we need a maximum for globally applicable widths?
+// i.e. .width-500 on a 414px wide screen would be a problem
+// do we need min and max? They are used sometimes.
+// But we're definitely generating more of every category of atomic than we need.
+
+// widths
+
+@each $width in $widths {
+	.width-#{$width} {
+		width: #{$width}px !important;
+	}
+}
+
+@include tablet {
+	@each $width in $widths {
+		.width-#{$width}-tablet {
+			width: #{$width}px !important;
+		}
+	}
+}
+
+@include desktop {
+	@each $width in $widths {
+		.width-#{$width}-desktop {
+			width: #{$width}px !important;
+		}
+	}
+}
+
+@include widescreen {
```

Comment:
I think all the `widescreen` rules can be removed. I doubt it would be any difference there knowing the widescreen breakpoint is 1800 px.

Reply:
We do have a use case of widescreen for the chat window in LearnTV,

Reply:
To me that particular instance could be either removed or changed to a slightly lower value or only applied on desktop.

Reply:
I would recommend changing the applied class on LearntTV to `-desktop` and don't have any `widescreen` helpers.

Reply:
Sounds like a good plan to me

Reply:
widescreen is gone

Reply:
🙌🏻🙌🏻🙌🏻